templates/snmpd.conf.erb

```diff
@@ -132,6 +132,8 @@ smuxpeer .1.3.6.1.4.1.674.10892.1
 #Allow Systems Management Data Engine SNMP Storageservices to connect to snmpd using SMUX
 smuxpeer .1.3.6.1.4.1.674.10893.1
 <% end -%>
+<% if @snmpd_config -%>
```

Comment:
Why is this required?  `snmpd_config` is an Array that defaults to `[]`.  I can't be `undef`.